fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java

```diff
@@ -1396,6 +1400,16 @@ public static Auth read(DataInput in) throws IOException {
         return auth;
     }
 
+    public static boolean isSupportKerberosAuth() {
+        if (!Config.authentication_kerberos_enabled ||
+                Config.authentication_kerberos_service_key_tab.isEmpty() ||
+                Config.authentication_kerberos_service_principal.isEmpty()) {
+            return false;
+        }
+
+        return new File(KRB5_AUTH_JAR_PATH).exists();
```

Comment:
This is a heavy operation.
is it necessary to be checked?

Reply:
IMO the JAR file need to be loaded only once. Could you optimize it?

Reply:
done